### ISSUE TITLE
Find Forms - "Must" Accessibility Items

### ISSUE THREAD

**Issue by MickinSahni** [department-of-veterans-affairs/va.gov-team#7096]
## User Story or Problem Statement
Given we want our new Find VA Forms feature to be available to people of every ability
I want to resolve "must" accessibility items we've identified. 

## Tasks

- [x]  Manage focus when users press Search. Ideally we would move focus to the "Showing results for `SEARCH TERM`"

- [x] Confirm we're not using `aria-live` region

- [x] Focus will need to be managed when we have pagination too. Probably set focus back to the same spot.
* ~"Apply online" button will need an aria-label or SR-only text to be clear to screen readers. For instance:- Not including links to online tools in updated mobile designs so this won't be necessary~

  ```html
  <a href="#">
    <span aria-hidden="true">Apply online</span>
    <span class="sr-only"Apply online for 10-10072 Supportive Services for Veteran Families</span>
  </a>
  ```

## Acceptance Criteria
- [x] All "must" accessibility findings are resolved

## Notes
Addresses Must findings here: https://github.com/department-of-veterans-affairs/va.gov-team/issues/6825

**Comment by MickinSahni** [comment]
2020-03-27 PW Standup
- In progress
- Q: where should focus be _while_ the results are loading—the loading indicator or the submit button? Granted, once results are loaded focus will go to "Displaying results...".
  - @jenstrickland could you help us out with what's most appropriate here?

**Comment by jenstrickland** [comment]
@MickinSahni I recommend:

1. Focus moves to an indication that work is in progress (ideally this is designed in a way that is empathetic to screen reader users as well as sighted users)
    1. For example, a visual loading indicator receives focus with a screen reader only message that we are processing your search query, thank you for your patience
    1. Alternatively, there could be text displayed that receives focus indicating the system is searching for their results that is visible to sighted users and read by screen readers.
1. When resulted are located and loaded, it would go to "Displaying results…"

**Comment by kelsonic** [comment]
Unassigning myself from this ticket, but ping me if you need anything! 🙂

**Comment by MickinSahni** [comment]
2020-03-30 PW Standup
- PR merged, should be able to validate today
- May be an issue related to hyphens on the "Displaying ...."

**Comment by MickinSahni** [comment]
2020-04-01 PW Standup
- This is related to #7097. Adding Jennifer as she's likely need to validate

**Comment by MickinSahni** [comment]
2020-04-02 PW Standup
- @jenstrickland left a comment on #7097 but it prob applies here—any other must items to be resolved?

**Comment by jenstrickland** [comment]
@MickinSahni - just checked this, and looks good. If a link to online tools is added, though, please keep the crossed out guidance in mind. 

**Thanks so much for quickly addressing these!**
